packages/create-redwood-app/templates/js/api/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
+    "@prisma/client": "*",
```

Comment:
Interesting, so even though `api` has the dependency https://github.com/redwoodjs/redwood/blob/220a7539573aba331735a207e0497abeb5cfa3e8/packages/api/package.json#L35 it doesn't "bubble up" per se to get completion.

Reply:
`@prisma/client` is a dependency of a dependency within the app, that means for TypeScript "don't show in auto-imports" - otherwise the auto-imports would include every possible dependency in your node_modules, and that's an impossibly big space to work with (and quite brittle as recommendations)

It will appear in a file where you have already imported that dep though, which is why it's generally not too annoying - because most API files have it in their templates